### PR TITLE
Add options to limit the send queue length

### PR DESCRIPTION
**Pull request by mrunge**
Collectd has the option to limit the send queue lenght
for amqp1. This patch adds support for that.

For reference, that was added in
https://github.com/collectd/collectd/pull/3432/files

### CODE REVIEW

**Comment by mrunge** [comment]
Does anyone have an idea why CI fails on this? I mean, I see a lot of "errors", but they are not related to this change?

**Comment by mrunge** [comment]
Is there a way to test this patch on top of another PR, like on https://github.com/voxpupuli/puppet-collectd/pull/951 ?

File: examples/plugins/filecount.pp [hunk 1]
```diff
@@ -7,11 +7,11 @@
   },
 }
 
-collectd::plugin::filecount::directory { 'foodir':
+collectd::plugin::filecount::directory {'foodir':
```
Comment (bastelfreak) [inline examples/plugins/filecount.pp:10]: I think out linter plugin requires the whitespace here?

Comment (mrunge) [reply examples/plugins/filecount.pp]: yes! I thought I had that commit removed. This has nothing to do with the original change and was just a try to make CI pass.

**Pull request merged**